libcontainer/console_linux.go

```diff
@@ -13,7 +12,7 @@ import (
 // NewConsole returns an initalized console that can be used within a container by copying bytes
 // from the master side to the slave that is attached as the tty for the container's init process.
 func NewConsole(uid, gid int) (Console, error) {
-	master, err := os.OpenFile("/dev/ptmx", syscall.O_RDWR|syscall.O_NOCTTY|syscall.O_CLOEXEC, 0)
+	master, err := os.OpenFile("/dev/pts/ptmx", syscall.O_RDWR|syscall.O_NOCTTY|syscall.O_CLOEXEC, 0)
```

Comment:
Fair enough. This change came from some testing I was doing (in the host, `/dev/ptmx` and `/dev/pts/ptmx` have different semantics which made me suspicious of the legacy path). Ultimately, it doesn't _really_ matter since in most cases we're setting up the `/dev/ptmx` symlink anyway.